lua/transfer/transfer.lua

```diff
@@ -183,9 +210,9 @@ function M.remote_scp_path(local_path)
   return nil
 end
 
--- get the remote path for rsync
--- @param local_path string
--- @return string
+---get the remote path for rsync
```

Comment:
What is the reason to change comment style?

Reply:
I am using Treesitter parser luap for lua docstrings and they are only highlighted when using `---`:
![image](https://github.com/user-attachments/assets/786696fe-7d42-483c-9eea-4d18ff36701e)

I can revert it if you want though

Reply:
This is a bug in your highlighting. Treesitter itself and other popular lua plugins use --space in their codebase.